opencti-worker/src/worker.py

```diff
@@ -250,32 +218,20 @@ def data_handler(  # pylint: disable=too-many-statements, too-many-locals
         previous_standard = data.get("previous_standard")
         self.api.set_previous_standard_header(previous_standard)
         # Execute the import
-        self.processing_count += 1
-        content = "Unparseable"
+        imported_items = 0
+        event_type = data["type"] if "type" in data else "bundle"
+        types = (
+            data["entities_types"]
+            if "entities_types" in data and len(data["entities_types"]) > 0
+            else None
+        )
         try:
-            event_type = data["type"] if "type" in data else "bundle"
-            types = (
-                data["entities_types"]
-                if "entities_types" in data and len(data["entities_types"]) > 0
-                else None
-            )
-            processing_count = self.processing_count
-            if self.processing_count == PROCESSING_COUNT:
-                processing_count = None  # type: ignore
             if event_type == "bundle":
                 content = base64.b64decode(data["content"]).decode("utf-8")
                 update = data["update"] if "update" in data else False
-                self.api.stix2.import_bundle_from_json(
-                    content, update, types, processing_count
+                imported_items = self.api.stix2.import_bundle_from_json(
```

Comment:
import_bundle_from_json returns a list of elements, not a number, and imported_items is initialized at 0. is it a list or a number ?

Reply:
Its a list, the initialization to 0 must be an empty array. Thanks